app/helpers/application_helper/title_methods.rb

```diff
@@ -0,0 +1,64 @@
+module ApplicationHelper::TitleMethods
+  include MarkdownHelper
+  include UserHelper
+
+  def generate_title(name, junction = nil, content = nil, s = false)
+    if s
+      if name[-1].downcase != "s"
+        name = name + "'s"
+      else
+        name = name + "'"
+      end
+    end
+
+    list = [name]
+
+    list.push junction unless junction.nil?
+
+    unless content.nil?
+      content = strip_markdown(content)
+      if content.length > 45
+        content = content[0..42] + "..."
```

Comment:
```suggestion
        content = content[0..42] + "…"
```

Reply:
Of course I will accept a suggestion that will make the tests fail!

Reply:
Oops 🙈